### PR TITLE
:robot: Add processhacker.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/565 :rocket:

### CODE REVIEW

File: packages/processhacker.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+  $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+  $executablePath = Join-Path ${Env:ProgramFiles} "\Process Hacker 2\ProcessHacker.exe" -Resolve
+  Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -RunAsAdmin
+  VM-Assert-Path $shortcut
+
+  # Delete Desktop shortcut
+  $desktopShortcut = Join-Path ${Env:Public} "Desktop\Process Hacker 2.lnk"
+  if (Test-Path $desktopShortcut) {
```
Comment (Ana06) [inline packages/processhacker.vm/tools/chocolateyinstall.ps1:16]: [nitpick] This `if` is not needed as the next command has the `-ea 0`that is the same as `-ErrorAction SilentlyContinue`. We have currently some packages with this if and some without. I would remove it for simplicity.

File: packages/processhacker.vm/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,21 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'ProcessHacker'
+  $category = 'Utilities'
+
+  $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+  $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+  $executablePath = Join-Path ${Env:ProgramFiles} "\Process Hacker 2\ProcessHacker.exe" -Resolve
```
Comment (Ana06) [inline packages/processhacker.vm/tools/chocolateyinstall.ps1:10]: Should we add the tool to path as well?
```
Install-BinFile -Name $toolName -Path $executablePath
```

**Review by Ana06 — COMMENTED** [review]
It was discussed in https://github.com/mandiant/VM-Packages/issues/161 to add System Informer instead of Process Hacker. Any reason you need Process Hacker (over System Informer)?

Can you please squash the commits or use "Squash and merge" when merging? We don't want to keep `fix` commits in order to have a clean commit history. I also recommend reading about commits good practices (Use imperative mood, capitalize the first word,  etc), for example:
- https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages
- https://about.gitlab.com/blog/2018/06/07/keeping-git-commit-history-clean/

**Comment by nos3curity** [comment]
Closing until further notice. Seems like System Informer might be the better option here.

**Pull request closed without merging**